scripts/mappings/swapuzMappings.ts

```diff
@@ -608,10 +608,10 @@ swapuz.set('THETA', null)
 swapuz.set('THORCHAIN', 'thorchainrune')
 
 // Display Name: TELOS
-swapuz.set('TLOS', null)
+swapuz.set('TLOS', 'telos')
 
 // Display Name: TELOS(EVM)
-swapuz.set('TLOSEVM', null)
+swapuz.set('TLOSEVM', 'telos')
```

Comment:
TLOS and TLOSEVM shouldn't point to the same pluginId, right? Also, TLOSEVM isn't support in Edge